client/src/components/BoulderCard.spec.jsx

```diff
@@ -0,0 +1,34 @@
+import BoulderCard from './BoulderCard';
+
+const Boulder = {
+    number: 1,
+    name: 'Black Widow and the people',
+    sector: 'Monkey Island',
+    level: '5',
+    handle_color: 'black',
+    setter: 'Martin Pagels',
+    tags: ['Dyno', 'Sloper'],
+    img_start: 'Boulder-girl.jpg',
+    img_all: 'imageAll.jpg',
+    weighting: 100,
+    number_of_likes: 5,
+    start_date: '2022-04-26',
+    expiry_date: '',
+    climbers: [],
+  }
```

Comment:
Your test data here also have keys that aren't used in your component.

Reply:
See my comment above.